test/js-native-api/test_object/test.js

```diff
@@ -208,12 +208,13 @@ assert.strictEqual(newObject.test_string, 'test string');
   // i.e.: includes prototypes, only enumerable properties, skips symbols,
   // and includes indices and converts them to strings.
 
+  const symbol = Symbol('foo')
   const object = Object.create({
     inherited: 1
   });
 
   object.normal = 2;
-  object[Symbol('foo')] = 3;
+  object[symbol] = 3;
```

Comment:
Can you set the object's prototype and set a property on that, and retrieve it below?